public/video-ui/src/services/capi.ts

```diff
@@ -71,18 +76,8 @@ export default class ContentApi {
   }
 
   static getTagsByType(query: string, types: string[]) {
```

Comment:
It doesn't makes sense for this to live in the ContentApi class, given we are querying Tag Manager.